packages/core/android/src/main/kotlin/com/datadog/reactnative/DdTraceImplementation.kt

```diff
@@ -6,18 +6,26 @@
 
 package com.datadog.reactnative
 
-import com.datadog.android.tracing.AndroidTracer
+import com.datadog.android.trace.AndroidTracer
+import com.datadog.android.trace.Trace
+import com.datadog.android.trace.TraceConfiguration
 import com.facebook.react.bridge.Promise
 import com.facebook.react.bridge.ReadableMap
 import io.opentracing.Span
 import io.opentracing.Tracer
+import io.opentracing.util.GlobalTracer
 import java.util.concurrent.TimeUnit
 
 /**
  * The entry point to use Datadog's Trace feature.
  */
 class DdTraceImplementation(
-    private val tracerProvider: () -> Tracer = { AndroidTracer.Builder().build() }
+    private val tracerProvider: () -> Tracer = {
+        val tracer = AndroidTracer.Builder().build()
+        GlobalTracer.registerIfAbsent(tracer)
```

Comment:
yes, it is still needed. We cannot move it behind `.enable` call as it was done for RUM, because `GlobalTracer` is not owned by us (it is coming from `io.opentracing`) and may be already used by the user for something else.